Development

robertmansour88 opened the pull request.

behoyh committed:
.gitignore

behoyh committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Development

robertmansour88 committed:
Added HTML Mobirize file

robertmansour88 committed:
Mobirise Update

robertmansour88 committed:
clergy update

robertmansour88 committed:
update gitignore

behoyh committed:
Fixed Index Links up till book a confession appointment

behoyh committed:
small fix

behoyh commented:
Alright Robert do the rest of the menu in index.html and I'll go ahead and review this

behoyh committed:
Updated index page

behoyh committed:
fixed text display in home on mobile

behoyh committed:
take /2

behoyh committed:
Footer fixes

behoyh committed:
fix mobile hamburger button

behoyh committed:
added tables to liturgies page

behoyh committed:
Fixed clergy page

behoyh committed:
text color fixes

robertmansour88 committed:
Clergy HTML modifications

just a quick tag layout reformat

robertmansour88 committed:
Navigation Menu HTML Updates

- Synced the Nav menu to all HTML files with proper links
- Added a drop down item 'Arabic Meeting' under services for Arabic
Meeting Calendar
- Created Arabic Meeting Calendar.html and linked to Nav item

robertmansour88 commented:
Can you approve the pull request. It has all recent updates.

behoyh reviewed (approved):
Approved!
